### ISSUE TITLE
win11  23H2 live tiles not de-bloated

### ISSUE THREAD

**Issue by rrattinypcbs** [gyp1jsi/MakeWindowsGreatAgain#8]
Just noting that live tiles were not un installed. Also many other apps failed to unload. Will add notes where I can when I get a few moments. /Rob

**Comment by gyp1jsi** [comment]
I will sort it out eventually in 1.3.1 or in 2.0.0 directly, the next major release. If you want, could you give me the appx package names of such live tiles?

**Comment by rrattinypcbs** [comment]
I will send soon. I will be building a few systems in a couple of weeks. I can build some 22H2 and some 23H2 and list them separately if that would be helpful...not sure it is unless they have different names right? /Rob

**Comment by gyp1jsi** [comment]
I already made the necessary changes for 22H2 removal some versions ago. Only 23H2 is necessary.

**Comment by rrattinypcbs** [comment]
Ok, so you need some 23H2 appx list? And what version handles 22H2, I must have missed the changelog. With kind regards, Rob

**Comment by gyp1jsi** [comment]
Yes, I just need 23H2 appx list. The version that handled 22H2 was one of the first ones, as the script was first released on Oct 1, 2022.
